common/app/mvt/MultiVariateTesting.scala

```diff
@@ -132,14 +117,7 @@ object MultiVariateTesting {
 
   sealed case class Variant(name: String)
 
-  // buckets 0-7 are removed because they cost $1000+ just in aws bandwidth every month, the rest
-  // will be removed once they're not in use.  In future server side ab tests will be added explicitly
-  // to target only the URLs needed for the time needed.
-  object Variant8 extends Variant("variant-8")
-  object Variant9 extends Variant("variant-9")
-
-  private val allVariants = List(
-    Variant8, Variant9)
+  private val allVariants = List[Variant]()
```

Comment:
@johnduffell I will follow up in the next few days with another PR to refactor and clean this `MultiVariateTesting`. I didn't want to mix all those changes here